### ISSUE TITLE
ScoreIteratorListener outputs same line during testfold

### ISSUE THREAD

**Issue by kaitoii11** [Medha-B/unetdl4j#7]
When I run LearningKfold.java, I get duplicate lines in the output.
This is probably caused by adding ScoreIteratorListener multiple times [here](https://github.com/Medha-B/unetdl4j/blob/83a363a0556bfa29a78240b53884e6f00ad9796e/src/main/java/org/sbml/spatial/segmentation/LearningKfold.java#L149).

```
23:50:29.120 [main] INFO org.deeplearning4j.optimize.listeners.ScoreIterationListener - Score at iteration 650 is 1157.8824896853794
23:50:29.120 [main] INFO org.deeplearning4j.optimize.listeners.ScoreIterationListener - Score at iteration 650 is 1157.8824896853794
23:50:29.120 [main] INFO org.deeplearning4j.optimize.listeners.ScoreIterationListener - Score at iteration 650 is 1157.8824896853794
```

**Comment by Medha-B** [comment]
Had not noticed this before but it is happening on my end too. Looking for possible solutions.

**Comment by kaitoii11** [comment]
As I mentioned in my earlier comment, the code adds `ScoreIteratorListener` within the for loop, which means that `ScoreIteratorListener` is added more than once. So adding `ScoreIteratorListener` outside the loop should fix this problem

**Comment by Medha-B** [comment]
Did the same. The code works fine now. Will make a pull request soon.

**Comment by Medha-B** [comment]
Kindly check the new commits to the branch modifyPath. I did not observe any repetition of ScoreIterationListener. Please let me know in case of any discrepancies.

**Comment by kaitoii11** [comment]
Ok, I'll check on my side.

Next time, you should create a diffrent branch when working on a different issue unless they are related.

**Comment by kaitoii11** [comment]
close #4

**Comment by kaitoii11** [comment]
You didn't have to manually close this issue.
This issue would have closed once pull request #6 was merged.
see this [link](https://docs.github.com/en/enterprise/2.16/user/github/managing-your-work-on-github/closing-issues-using-keywords) for more details.
